shared/js/project.js

```diff
@@ -125,6 +135,12 @@ var projects = module.exports = {
     }
   },
 
+  showAdmin: function() {
+    if (typeof dashboard.admin.showProjectAdmin === 'function') {
```

Comment:
When would `dashboard.admin.showProjectAdmin` not exist?  (I.e. what is this `if` check defending against?)